### ISSUE TITLE
Bookmark elements are not keyboard navigable

### ISSUE THREAD

**Issue by seanaery** [projectblacklight/blacklight#3555]
In either a search results/index view or an item show page, a user can't currently reach the bookmark element via keyboard to check or uncheck it. Tabbing through the UI, the bookmark checkbox never receives focus.

<img width="792" alt="Image" src="https://github.com/user-attachments/assets/65e34dbd-96ba-45f1-bb21-4572906ccbbe" />
